gazebo_usar_core/ChemicalSubstance[] chemicalSubstances
